web/src/main/java/com/epam/rft/atsy/web/mapper/RuleValidationExceptionMapperImpl.java

```diff
@@ -1,25 +1,37 @@
 package com.epam.rft.atsy.web.mapper;
 
+import com.epam.rft.atsy.web.configuration.ExceptionMapperConfiguration;
 import org.springframework.stereotype.Component;
 import org.springframework.util.Assert;
 
 import java.util.Set;
 import javax.annotation.Resource;
 
+/**
```

Comment:
Follow Javadoc conventions: write the first sentence as a short summary of the method, as Javadoc automatically places it in the method summary table (and index).